### DISCUSSION TITLE
Ended gitHub copilot subscription

### DISCUSSION THREAD

**Discussion by Luchoragusa** [community/community#39532] [GitHub Education]
Hi, I'm a student and I'm approved for the GitHub Student package, but today I received a notification that my subscription to Copilot has ended, but for students it's free.

**Comment by a-type** [comment]
I'm a paying, subscribed user and my CoPilot is also showing this notification and won't connect today.

**Reply by a-type** [reply]
Update: seems resolved now.

**Reply by stevehansen** [reply]
stopped working for me, always says I don't have access

**Reply by Digital-Coder** [reply]
yep, stopped working for me too

**Reply by ecyrbe** [reply]
stopped working 10 minutes ago for me

**Reply by jwestern1337** [reply]
same here

**Reply by helderpgoncalves** [reply]
yep. Seems general bug

**Reply by Sohaib-GO** [reply]
stopped

**Reply by ramyatrouny** [reply]
same here

**Reply by mchaniotakis** [reply]
every month im having the same problem!

**Reply by anhvth** [reply]
> every month im having the same problem!

Could you let me know how long does it take to fixed the problem?

**Comment by bogdanws** [comment]
I have the same issue, I was in the middle of coding when Copilot suddenly notified me that my subscription has ended, even though I'm approved for the GitHub Student package.

**Comment by LandryFio** [comment]
Exactly the same issue here.

**Reply by LandryFio** [reply]
UPDATE : Seems resolved for me now.

**Reply by hjx159** [reply]
> 更新：现在对我来说似乎解决了。
How did you solve it?

**Reply by ramyatrouny** [reply]
how to solve it?

**Comment by risan-raja** [comment]
same issue

**Reply by risan-raja** [reply]
Did Anyone Find a solution? @github-staff  Please help us.

**Reply by LandryFio** [reply]
> Did Anyone Find a solution? @github-staff Please help us.

In my case it was automatically fixed by restarting VS Code.

**Reply by risan-raja** [reply]
@LandryFio I tried raising a ticket and it got closed with some generic reply.

**Reply by OneAndOnlyFinbar** [reply]
The issue resolved itself yesterday, at least for me. Maybe give it some time.

**Reply by rgprom** [reply]
Happening still for me too, even after a restart. Also, do I really need the $4 pro upgrade to use co-pilot? They started charging me for $4 pro + $10 co-pilot.

**Reply by ethanclawsie** [reply]
Same here.

**Comment by samus-aran** [comment]
Hey folks 👋 Glad to hear most of yous have it resolved.

Who is still having issues with not accessing Copilot although still a verified student?

**Reply by LisraCalco** [reply]
Hey got mine canceled just now, it show in billing I have access as student but copilot area show it as "ended"
Help would be appreciated, Thanks :D

**Reply by LandryFio** [reply]
Mine got also cancelled once again... right now

**Reply by stevehansen** [reply]
paying customer and it also shows Extended preview access has ended, says I don't have access

```
GitHub Copilot 1.61.0.0 (v1.61.0.0@5e1cdbba5)
GitHub Copilot Agent v1.61.0
You don't have access to GitHub Copilot.
Please open https://github.com/login/device and enter the device activation code XXXX-XXXX that has been copied to the clipboard.
You don't have access to GitHub Copilot.
```

**Reply by EduardoArtioli** [reply]
Please fix this.

**Reply by ethanclawsie** [reply]
Same here.

**Reply by Digital-Coder** [reply]
My yearly plan is not working and it throws "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."

**Reply by Mai0313** [reply]
me T^T I am still a student but its disabled

**Reply by ignacio-urrutia** [reply]
Me!

**Reply by senecamanu** [reply]
I got monthly plan and it still shows: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended." I think it's been an hour.

**Reply by damag-dev** [reply]
Same here . Not working for verified student , it shows "Your subscription has ended."

**Reply by AllanDeuz** [reply]
I still have problems with copilot /:, I'm a student and it shows "Your subscription has ended."

**Reply by ddkvdk** [reply]
Me

**Reply by beba42** [reply]
i still have an issue too

**Reply by ahmadsalimi** [reply]
I still have the issue

**Reply by koteelok** [reply]
I have an issue

**Comment by paul-plaza** [comment]
Exactly the same issue here.

**Reply by ethanclawsie** [reply]
Same here.

**Comment by ethanclawsie** [comment]
Same here.

**Comment by mxdi9i7** [comment]
I have an active annual subscription and i just got the same message...

**Reply by ramyatrouny** [reply]
same here, were you able to solve it or not yet?

**Comment by nuuritas** [comment]
Same here. It happened a couple of weeks ago, and similar to all of you happened again half an hour ago.

**Comment by System10111** [comment]
I have the same issue. I am a verified student, and on the billing and plans page it says "You are eligible to use GitHub Copilot for free", but on the copilot page it says "Your subscription has ended".

**Reply by ghost** [reply]
same...

**Comment by 25pinjo15** [comment]
same ...

**Comment by lsrweb** [comment]
Same problem, my Github copilot cannot be used. Pro Version...

**Comment by johnalbert-dot-py** [comment]
Just happen to me right now, I'm still verified as a student on my github student developer pack

**Comment by qhduan** [comment]
same here

I'm not student, I have set my billing account, but still give me 'Your subscription has ended.'

**Reply by anhvth** [reply]
same

**Comment by teoprayoga** [comment]
same here, I'm a student with pro labeled. and why the message 'Your subs has ended'?

**Comment by ddkvdk** [comment]
Same problem

**Comment by rgprom** [comment]
The fix was this for me: I was logged into VS Code with several github accounts. So, I had to log out of all of them, except for the one I had the subscription to. Then I restarted VS Code and it worked! I might've also had to uninstall/reinstall co-pilot. Thanks!

**Reply by ramyatrouny** [reply]
but it shows that the GitHub copilot subscription ended from the web portal.

**Comment by ThanksOihyuo** [comment]
> Hi, I'm a student and I'm approved for the GitHub Student package, but today I received a notification that my subscription to Copilot has ended, but for students it's free.

**Comment by ramyatrouny** [comment]
I already paid for a 1-year subscription but surprisingly it shows ended.

**Reply by helderpgoncalves** [reply]
This is a bug, maybe there are updating the feature because it happens to me 2 weeks ago but was solved the next day automatically.

**Comment by harshverma250202** [comment]
same here

**Comment by koteelok** [comment]
same here

**Comment by otkrickey** [comment]
I have created a discussion for today's case. Please share your situation. #43634

**Comment by davidlechuga** [comment]
i have the answer, just change the version and reload the window of VScode.

![7](https://user-images.githubusercontent.com/47370612/227614667-d3a93143-16ff-45a9-933d-9f355e93dc10.png)
